firefox-ios/firefox-ios-tests/Tests/SyncIntegrationTests/Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "5c8b5b678f28852600d42fccc45bc6387f68bccea6613819686d72b66ef7b1f2"
+            "sha256": "710b2b26f95df49fcc12cc73f8922fe70dca0ebb74636b2099b189499bdb9391"
```

Comment:
If we don't like these changes, I can roll back everything except the ones related to `pytest-fxa-mte` and `pyfxa-mte`. What do you all think?

Since Pipfile does not pin 📌 particular versions of depdencies, `pipenv install` bumps the versions.

Reply:
That probably makes sense, you only want changes that reflect changes in this commit.